tier3-icons.html

```diff
@@ -136,11 +142,8 @@
           <path d="M7,2,25,15,7,28V2M7,0A2,2,0,0,0,5,2V28a2,2,0,0,0,3.17,1.62l18-13a2,2,0,0,0,0-3.24l-18-13A2,2,0,0,0,7,0Z"/>
         </g>
         <g id="play">
-          <path d="M26,2c1.103,0,2,0.897,2,2v22c0,1.103-0.897,2-2,2H4c-1.103,0-2-0.897-2-2V4c0-1.103,0.897-2,2-2H26 M26,0H4
-	C1.8,0,0,1.8,0,4v22c0,2.2,1.8,4,4,4h22c2.2,0,4-1.8,4-4V4C30,1.8,28.2,0,26,0L26,0z"/>
-          <path d="M12,9l9,6l-9,6V9 M12,7c-0.324,0-0.648,0.078-0.944,0.237C10.406,7.585,10,8.262,10,9v12
-	c0,0.738,0.406,1.415,1.056,1.763C11.352,22.922,11.676,23,12,23c0.388,0,0.775-0.113,1.11-0.336l9-6C22.666,16.293,23,15.669,23,15
-	s-0.334-1.293-0.891-1.664l-9-6C12.775,7.113,12.388,7,12,7L12,7z"/>
+          <path d="M26,2c1.103,0,2,0.897,2,2v22c0,1.103-0.897,2-2,2H4c-1.103,0-2-0.897-2-2V4c0-1.103,0.897-2,2-2H26 M26,0H4&#xA;&#x9;C1.8,0,0,1.8,0,4v22c0,2.2,1.8,4,4,4h22c2.2,0,4-1.8,4-4V4C30,1.8,28.2,0,26,0L26,0z"/>
```

Comment:
Same with these.  These are line feed characters.  Do we care if these are here?

Reply:
Happy to make whatever changes are necessary. But again, these were also part of the output of `npm run build`.

Reply:
Yeah ideally I think we don't want them.  I'm just not too sure why there are there now.

Reply:
Didn't Justin's commit remove the line feed characters?

Reply:
Hmm yeah.  Maybe if last person didn't use generated, then it explains it.  :)